h/accounts/test/views_test.py

```diff
@@ -830,30 +829,28 @@ def test_activate_event_when_validation_succeeds(event,
 
 
 @pytest.mark.usefixtures('subscriptions_model')
-def test_profile_looks_up_by_logged_in_user(authn_policy, user_model):
-    """
-    When fetching the profile, look up email for the logged in user.
+def test_profile_looks_up_by_logged_in_user():
```

Comment:
It was testing that the correct arguments were passed to `get_by_userid`. `.authenticated_user` has no signature so there's no collaboration to test. Put another way, if this function passed all the other tests, but failed this one because you'd refactored it to get the user another way, that would be silly.